tests/java_helics/CMakeLists.txt

```diff
@@ -140,6 +140,7 @@ set_property(
     LABELS
     Java
     Continuous
+    Coverage
```

Comment:
I doubt the Java, matlab, python test will add much on the coverage side, so wondering if this is necessary?  Are we checking coverage of the python/ Java code?

Reply:
No, we aren't checking coverage of the Python/Java code. I think the only contribution it adds now is if those interfaces use some part of the shared library that the shared library test cases don't cover.

Reply:
I think I would prefer knowing if the shared_library tests didn't cover a particular aspect of the C library.  @kdheepak any thoughts?